### PR TITLE
Branch out android platform view perf testing

### PR DESCRIPTION
**Pull request by cyanglaz**
## Description

The embedded platform view on android requires a flag to be set in the Manifest.xml after https://github.com/flutter/engine/pull/19242
This flag will result a static thread merging.
We want to keep the virtual display platform view on android still runs on the raster thread.
So this PR branches out the embedded platform view test on android to a different app.

we should wait unitl https://github.com/flutter/engine/pull/19242 and https://github.com/flutter/engine/pull/19257 rolled into the framework then land this, otherwise the app will stuck.

## Related Issues

https://github.com/flutter/flutter/issues/60016

## Tests

benchmarks/platform_views_layout_hybrid_composition

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
   - [ ] I wrote a design doc: https://flutter.dev/go/template *Replace this with a link to your design doc's short link*
   - [ ] I got input from the developer relations team, specifically from: *Replace with the names of who gave advice*
   - [ ] I wrote a migration guide: https://flutter.dev/go/breaking-changes-template *Replace this with a link to a pull request that adds the migration guide to https://flutter.dev/docs/release/breaking-changes*

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: dev/benchmarks/platform_views_layout_hybrid_composition/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                  android:hardwareAccelerated="true"
+                  android:windowSoftInputMode="adjustResize">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN"/>
+                <category android:name="android.intent.category.LAUNCHER"/>
+            </intent-filter>
+        </activity>
+        <meta-data
+            android:name="io.flutter.embedded_views_preview"
+            android:value="true" />
```
Comment (blasten) [inline dev/benchmarks/platform_views_layout_hybrid_composition/android/app/src/main/AndroidManifest.xml:24]: 👍

File: dev/devicelab/lib/tasks/perf_tests.dart [hunk 2]
```diff
@@ -44,15 +44,15 @@ TaskFunction createUiKitViewScrollPerfTest() {
...
     testDriver: 'test_driver/scroll_perf_test.dart',
   ).run;
 }
 
 TaskFunction createAndroidViewScrollPerfTest() {
   return PerfTest(
-    '${flutterDirectory.path}/dev/benchmarks/platform_views_layout',
+    '${flutterDirectory.path}/dev/benchmarks/platform_views_layout_hybrid_composition',
     'test_driver/android_view_scroll_perf.dart',
     'platform_views_scroll_perf',
```
Comment (blasten) [inline dev/devicelab/lib/tasks/perf_tests.dart:57]: this should be `platform_views_scroll_perf_hybrid_composition` now.  Maybe: `platform_views_hybrid_composition_scroll_perf`

Comment (cyanglaz) [reply dev/devicelab/lib/tasks/perf_tests.dart:57]: does the test file name matter? The root directory is already different. (now `platform_views_layout_hybrid_composition`)

**Review by blasten — APPROVED** [review]
LGTM

**Pull request merged**